sound/soc/sof/sof-priv.h

```diff
@@ -301,7 +301,7 @@ struct snd_sof_dfsentry {
 	 * or if it is accessible only when the DSP is in D0.
 	 */
 	enum sof_debugfs_access_type access_type;
-#if ENABLE_DEBUGFS_CACHEBUF
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_ENABLE_DEBUGFS_CACHE)
```

Comment:
separate/unrelated patch?

Reply:
@plbossart this isnt an unrelated change. We no longer use the snd_sof_dfsentry for flood test debugfs and the DEBUG_IPC_FLOOD_TEST has been removed. So this is a valid change in this patch. WHats missed is removing the macro ENABLE_DEBUGFS_CACHEBUF altogether.